app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class UserSerializer < ActiveModel::Serializer
+  attributes :id, :name, :email, :following_count, :follower_count, :article_count, :gender, :username
```

Comment:
Since this is becoming very long, you can write it as:

```ruby
attributes (
    :id,
    :name,
    :email, 
    :following_count, 
    :follower_count, 
    :article_count, 
    :gender, 
    :username
)